internal/core/ports/ocean_wallet.go

```diff
@@ -0,0 +1,121 @@
+package ports
+
+import "context"
+
+type OceanWallet interface {
```

Comment:
In general core/ports should not be aware about its implementation, in this case ocean, as tdexd doesnt care about it all it cares is that it is using Wallet interfaces with all its methods, so i would rename file to wallet_service or smtn.
Also i think we should decide to use one name for types/files and instead service/manager we should pick one, my favourite is Service, wtf is Manager :)
So in this case I would name file wallet_service, main interface would be WalletService and other intf would be Wallet, Account, Transaction etc

Reply:
done.